x/validator-vesting/client/rest/query.go

```diff
@@ -0,0 +1,102 @@
+package rest
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/cosmos/cosmos-sdk/types/rest"
+
+	"github.com/kava-labs/kava/x/validator-vesting/types"
+)
+
+func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
+	r.HandleFunc("/vesting/circulatingsupply", getCirculatingSupplyHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/vesting/totalsupply", getTotalSupplyHandlerFn(cliCtx)).Methods("GET")
```

Comment:
An explicit route variable would be clearer, declare const "vesting". May avoid some confusion as the other modules all use `/{ModuleName}/path/to/endpoint`, this is the only one that doesn't use the module name.

Reply:
resolved by https://github.com/Kava-Labs/kava/pull/586/commits/12a904214881a7ddf80ebd465c7f49674aaba68e